src/helper-text/index.tsx

```diff
@@ -1,42 +1,32 @@
-import { WidgetBase } from '@dojo/framework/core/WidgetBase';
-import { theme, ThemedMixin } from '@dojo/framework/core/mixins/Themed';
+import theme from '@dojo/framework/core/middleware/theme';
+import { create, tsx } from '@dojo/framework/core/vdom';
 import * as css from '../theme/default/helper-text.m.css';
-import { v } from '@dojo/framework/core/vdom';
-import { VNode } from '@dojo/framework/core/interfaces';
 
 export interface HelperTextProperties {
 	text?: string;
 	valid?: boolean;
 }
 
-@theme(css)
-export default class HelperText extends ThemedMixin(WidgetBase)<HelperTextProperties> {
-	protected render(): VNode {
-		const { text, valid } = this.properties;
+const factory = create({ theme }).properties<HelperTextProperties>();
 
-		return v(
-			'div',
-			{
-				key: 'root',
-				classes: this.theme([
-					css.root,
-					valid === true ? css.valid : null,
-					valid === false ? css.invalid : null
-				])
-			},
-			[
-				text &&
-					v(
-						'p',
-						{
-							key: 'text',
-							classes: this.theme(css.text),
-							'aria-hidden': 'true',
-							title: text
-						},
-						[text]
-					)
-			]
-		);
-	}
-}
+export default factory(function HelperText({ properties, middleware: { theme } }) {
+	const { text, valid } = properties();
+	const themedCss = theme.classes(css);
+
+	return (
+		<div
+			key="root"
+			classes={[
+				themedCss.root,
+				valid === true ? themedCss.valid : null,
+				valid === false ? themedCss.invalid : null
+			]}
+		>
+			{text && (
+				<p key="text" classes={themedCss.text} aria-hidden={'true'} title={text}>
```

Comment:
this can just be `aria-hidden="true"`. Did you get to the bottom of wether or not this is appropriate?